x-pack/test/observability_functional/apps/observability/alerts/workflow_status.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default ({ getService }: FtrProviderContext) => {
+  const esArchiver = getService('esArchiver');
+
+  describe('alert workflow status', function () {
+    this.tags('includeFirefox');
+
+    const observability = getService('observability');
+    const retry = getService('retry');
+
+    before(async () => {
+      await esArchiver.load('x-pack/test/functional/es_archives/observability/alerts');
+      await observability.alerts.navigateToTimeWithData();
+    });
+
+    after(async () => {
+      await esArchiver.unload('x-pack/test/functional/es_archives/observability/alerts');
+    });
+
+    it('is filtered to only show "open" alerts by default', async () => {
+      await retry.try(async () => {
+        const tableRows = await observability.alerts.getTableCellsInRows();
+        expect(tableRows.length).to.be(12);
+      });
+    });
+
+    it('can be set to "acknowledged" using the row menu', async () => {
```

Comment:
Nit: Similar here, this test name is still a bit unclear with regards to **what** "can be set". Maybe the word alert is needed here as well? 
"an alert can be set to "acknowledged" using the row action menu"? 

I don't think this needs to be changed, more looking to discuss patterns.